interfaces/builtin/joystick.go

```diff
@@ -82,15 +87,26 @@ var joystickConnectedPlugUDev = []string{
 	`KERNEL=="full", SUBSYSTEM=="mem"`,
 }
 
+type joystickInterface struct {
+	commonInterface
+}
+
+func (iface *joystickInterface) UDevConnectedPlug(spec *udev.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
+	spec.TriggerSubsystem("input/joystick")
```

Comment:
Come to think of it, perhaps something to add to commonInterface if we need to do this more often.